data_steward/requirements.txt

```diff
@@ -16,13 +16,13 @@ entrypoints==0.3
 enum34==1.1.6
 flask==0.10
 funcsigs==1.0.2
-google-api-core==1.17.0
+google-api-core==1.16.0
 google-api-python-client==1.6.4
-google-auth>=1.9.0
+google-auth==1.11.0
 google-auth-httplib2==0.0.3
 google-auth-oauthlib==0.4.1
 google-cloud-bigquery==1.24.0
-google-cloud-core>=1.1.0
+google-cloud-core==1.2.0
```

Comment:
Do you have a preference for pinning these to a specific version?

Reply:
Yes, I found that the latest versions may not be compatible with the surrounding packages which are pinned. We already have a ticket to upgrade the requirements file so I thought this could be a stop gap until then.

Reply:
Forget this comment for now.